syntax = "proto3";
package proto15test;

service FileTransferService {
    rpc Case1(FileRequest) returns (FileResponse);
    rpc Case2(stream FileRequest) returns (FileResponse);
    rpc Case3(FileRequest) returns (stream FileResponse);
    rpc Case4(stream FileRequest) returns (stream FileResponse);
}
message FileRequest {
    bytes image = 1;
    string filename = 2;
}
message FileResponse {
    string message = 1;
}


